### ISSUE TITLE
Edit the alarm "pop-up"?

### ISSUE THREAD

**Issue by tonynogs** [excaliburpartners/HAILogger#66]
Hi,

Does anyone know if it's possible to edit the Alarm "pop-up"?  I would like to remove the "Night", "Vacation", and "Custom" options from the pop-up. Thank you.

![Screenshot 2024-05-27 134828](https://github.com/excaliburpartners/OmniLinkBridge/assets/162737403/f73dcaa8-a8d4-40b0-b2b7-7dee4af93ff6)

**Comment by jlipsit** [comment]
> Hi,
> 
> Does anyone know if it's possible to edit the Alarm "pop-up"? I would like to remove the "Night", "Vacation", and "Custom" options from the pop-up. Thank you.
> 
> ![Screenshot 2024-05-27 134828](https://private-user-images.githubusercontent.com/162737403/334209881-f73dcaa8-a8d4-40b0-b2b7-7dee4af93ff6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4NDgwMjMsIm5iZiI6MTcxNjg0NzcyMywicGF0aCI6Ii8xNjI3Mzc0MDMvMzM0MjA5ODgxLWY3M2RjYWE4LWE4ZDQtNDBiMC1iMmI3LTdkZWU0YWY5M2ZmNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyN1QyMjA4NDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT00ZDkyMmYwYzQ2MDIyM2U0N2FlYzdmZjhjMGJjNjdhYmRmYmUxYTBiOWExMDAyYWFjNjExODE0ZTMwYWM1MzRlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.Tz9afqv-e_9fE_m4-43SnHaCHkFdlMXT7S0xh3DwWAs)

Yes, once you move it to a dashboard, you edit it and you can easily select what you want it to display. 

![IMG_3010](https://github.com/excaliburpartners/OmniLinkBridge/assets/155367039/e96140e0-f503-4721-a071-df19944f52e1)

**Comment by tonynogs** [comment]
But if you click on "DISARM", it opens up a pop up that shows all the options, right? It's the pop-up that I want to change.

**Comment by tonynogs** [comment]
So, I think I was unclear on what I'm looking for, and not sure this is something that can be answered here...I was to "edit out" the options "night", "vacation", and "custom" from the "more info" options. My mobile dashboard looks like this: 

![panel](https://github.com/excaliburpartners/OmniLinkBridge/assets/162737403/edc73b7c-ac04-42cc-9ed0-be69596a02b5)

When I click on the "Security" mushroom button, a pop up opens up (more info) that allows me to arm or disarm the system. Looks like this:

![no alarm](https://github.com/excaliburpartners/OmniLinkBridge/assets/162737403/bd3cf0c0-6c8f-4e4b-b645-040810924d9e)

BUT, what I really want is only the  options of "disarm:",  "away" or "home". Like this:

![yes alarm](https://github.com/excaliburpartners/OmniLinkBridge/assets/162737403/24ceb99a-1009-479a-b47a-e8814fcdaa31)

But I can't figure out how to disable "night", "vacation", and "custom" . Hope this makes sense? It might require me editing some yaml code somewhere, but for the life of me I can't figure out where. Thanks!

**Comment by rwagoner** [comment]
I plan to release 1.1.19 tomorrow with a `mqtt_discovery_override_area` option to allow customizing the modes that are shown. This version also fixes the alarm control panel compatibility issue with Home Assistant 2024.6. If you are using docker I pushed this build to the dev tag for testing.

**Comment by Burketj482** [comment]
Will the upcoming 1.1.19 release fix this problem I now have with HA 2024.6 when I try to use the alarm control panel: Failed to call service alarm_control_panel/alarm_arm_away. Validation error: Arming requires a code but none was given for alarm_control_panel.area_1.  Thanks for any feedback.

**Comment by rwagoner** [comment]
> Will the upcoming 1.1.19 release fix this problem I now have with HA 2024.6 when I try to use the alarm control panel: Failed to call service alarm_control_panel/alarm_arm_away. Validation error: Arming requires a code but none was given for alarm_control_panel.area_1. Thanks for any feedback.

Yes, that is correct.

**Comment by Burketj482** [comment]
👍 Thank you.Sent from my iPhoneOn Jun 7, 2024, at 11:53 AM, Ryan Wagoner ***@***.***> wrote:﻿

Will the upcoming 1.1.19 release fix this problem I now have with HA 2024.6 when I try to use the alarm control panel: Failed to call service alarm_control_panel/alarm_arm_away. Validation error: Arming requires a code but none was given for alarm_control_panel.area_1. Thanks for any feedback.

Yes, that is correct.

**Comment by rwagoner** [comment]
Version 1.1.19 has been released with the fix for Home Assistant 2024.6 and the ability to change the arming modes displayed.

**Comment by Burketj482** [comment]
Got it! Works great! Thanks so much for working on this.TedSent from my iPhoneOn Jun 8, 2024, at 1:19 PM, Ryan Wagoner ***@***.***> wrote:﻿
Version 1.1.19 has been released with the fix for Home Assistant 2024.6 and the ability to change the arming modes displayed.

**Comment by enry86cami** [comment]
Hi, I updated to latest version and added this line:
mqtt_discovery_override_area = id=2; arm_home=false; arm_vacation=false; arm_night=false;
My goal is remove arm_home, arm_vacation, arm_night in area 2.
Is the line correct?

Thanks

**Comment by tonynogs** [comment]
> Version 1.1.19 has been released with the fix for Home Assistant 2024.6 and the ability to change the arming modes displayed.

Edit: got it working! Just a couple notes: 

1. For me it required a reboot of the device, just restarting HA didn't do the trick. 
2. You need to put a semi-colon between the items, not a comma as the documentation says.

Thank you!!!

![Screenshot 2024-06-15 181034](https://github.com/excaliburpartners/OmniLinkBridge/assets/162737403/c19f4fa9-2767-425b-b253-d478f44406d7)


![Screenshot 2024-06-14 205922](https://github.com/excaliburpartners/OmniLinkBridge/assets/162737403/ecac7a9f-664a-4dc0-ae5b-5bbe63113043)
